resources/zib/ext-FreedomRestrictingIntervention.LegallyCapable.xml

```diff
@@ -0,0 +1,105 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-freedomrestrictingintervention-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/FreedomRestrictingIntervention-v1.0(2020EN)" />
+    <name value="zib FreedomRestrictingIntervention-v1.0(2020EN)" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Procedure" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension.extension">
+      <path value="Extension.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Extension.extension:legallyCapable">
+      <path value="Extension.extension" />
+      <sliceName value="legallyCapable" />
+      <min value="1" />
+      <max value="1" />
+    </element>
+    <element id="Extension.extension:legallyCapable.url">
+      <path value="Extension.extension.url" />
+      <fixedUri value="legallyCapable" />
+    </element>
+    <element id="Extension.extension:legallyCapable.value[x]">
+      <path value="Extension.extension.value[x]" />
+      <short value="LegallyCapable" />
+      <definition value="Indicator stating whether the patient is deemed capable of overseeing the consequences of a certain procedure, situation or decision." />
+      <alias value="Wilsbekwaam" />
+      <min value="1" />
+      <type>
+        <code value="boolean" />
+      </type>
+      <mapping>
+        <identity value="zib-freedomrestrictingintervention-v1.0-2020EN" />
+        <map value="NL-CM:14.6.2" />
+        <comment value="LegallyCapable" />
+      </mapping>
+    </element>
+    <element id="Extension.extension:legallyCapableComment">
+      <path value="Extension.extension" />
+      <sliceName value="legallyCapableComment" />
+      <min value="1" />
+      <max value="1" />
+    </element>
+    <element id="Extension.extension:legallyCapableComment.url">
+      <path value="Extension.extension.url" />
+      <fixedUri value="legallyCapableComment" />
+    </element>
+    <element id="Extension.extension:legallyCapableComment.value[x]">
+      <path value="Extension.extension.value[x]" />
+      <short value="LegallyCapableComment" />
+      <definition value="Comment on the decisions which the patient is legally capable of making." />
+      <alias value="WilsbekwaamToelichting" />
+      <min value="1" />

```

Comment:
Waarom is dit (en  `Extension.extension`) verplicht gesteld? Het zib concept is 0..1 met weliswaar een constraint dat dit veld concept verplicht is als de LegallyCapable boolean false is. Volgens mij zou hier en FHIRpath beter op zijn plek zijn.

Reply:
Er is een tekstuele verplichting vanuit de zib: "If the patient is not legally capable, an explanation must be provided". Dit had ik iets te enthousiast naar een kardinaliteit van 1..1 vertaald. Ik pas het aan naar een constraint.